src/block/icon-button/style.scss

```
@@ -0,0 +1,18 @@
+@import "common";
+
+// We have a block group, and that has the bottom margin.
+.stk-block-icon-button {
+	--stk-block-margin-bottom: 0;
+}
+
+.stk-block.stk-block-icon-button {
+	width: auto;
+	margin: 0;
+	text-align: center;
+
+	a.stk-button {
+		// Make the buttons equal in height and width by default.
+		padding: 12px;
+		border-radius: 24px;
```

We can make this `100%` so that the block remains circular when adjusting the icon size
```suggestion
		border-radius: 100%;
```